### PR TITLE
update appsec rules label and description

### PR DESCRIPTION
**Pull request by AlteredCoder**

### CODE REVIEW

File: appsec-rules/crowdsecurity/vpatch-CVE-2021-3129.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: crowdsecurity/vpatch-CVE-2021-3129
-description: "Detect CVE-2021-3129 exploits "
+description: "Laravel with Ignition <= v8.4.2 Debug Mode - RCE (CVE-2021-3129)"
```
Comment (buixor) [inline appsec-rules/crowdsecurity/vpatch-CVE-2021-3129.yaml:2]: maybe a bit too long ? "Laravel with Ignition Debug mode RCE" ?

File: appsec-rules/crowdsecurity/vpatch-CVE-2022-46169.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: crowdsecurity/vpatch-CVE-2022-46169
-description: "Detect CVE-2022-46169 exploits "
+description: "Cacti <=1.2.22 - Remote Command Injection (CVE-2022-46169)"
```
Comment (buixor) [inline appsec-rules/crowdsecurity/vpatch-CVE-2022-46169.yaml:2]: "Cacti RCE" ?

File: appsec-rules/crowdsecurity/vpatch-CVE-2023-34362.yaml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 
 name: crowdsecurity/vpatch-CVE-2023-34362
-description: "Detect CVE-2023-34362 exploits "
+description: "MOVEit Transfer - Remote Code Execution (CVE-2023-34362)"
```
Comment (buixor) [inline appsec-rules/crowdsecurity/vpatch-CVE-2023-34362.yaml:3]: MOVEit Transfer RCE ?

File: appsec-rules/crowdsecurity/vpatch-CVE-2023-50164.yaml [hunk 4]
```diff
@@ -1,5 +1,5 @@
 name: crowdsecurity/vpatch-CVE-2023-50164
-description: "Detect CVE-2023-50164 exploits "
+description: "Apache Struts2 (CVE-2023-50164)"
```
Comment (buixor) [inline appsec-rules/crowdsecurity/vpatch-CVE-2023-50164.yaml:2]: Apache struts2 path traversal ?

**Review by buixor — CHANGES_REQUESTED** [review]

**Review by blotus — APPROVED** [review]

**Review by buixor — APPROVED** [review]

**Pull request merged**